test/unit/uri.c

```diff
@@ -1,3 +1,4 @@
+#define UNIT_TAP_COMPATIBLE 1
```

Comment:
Nit: There are commits that are not link any issue, but they're obviously made in a scope of #3682. I would mark them as 'Follows up #3682' or similarly.